voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md

```diff
@@ -0,0 +1,125 @@
+{% method %}
+
+## Download Transcription
+
+Retrieve the specified recording's transcription file. [⚠️ Be sure to not expose your API Credentials to end-users](./about.md#caution-recordings)
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}/transcription`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter | Description |
+|:----------|:------------|
+| None      | None        |
+
+{% common %}
+
+### Example 1 of 1: Download a transcription
+
+{% sample lang="http" %}
+
+```bash
+curl -X GET \
+    --url "https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}/transcription" \
+    -u '{username}:{password}'
+```
+
+```
+{
```

Comment:
do we need to document the output schema?

Reply:
I'm adding a description of the output we currently have, but please note that the output schema changes depending on the transcription options we use.
For instance, we don't use the channel identification feature or the speaker identification feature, so if we use that in the future, the schema would change.

Reply:
Examples of how the schema changes with each feature:
channel identification: https://docs.aws.amazon.com/transcribe/latest/dg/how-channel-id.html
speaker identification: https://docs.aws.amazon.com/transcribe/latest/dg/how-diarization.html
alternative transcriptions: https://docs.aws.amazon.com/transcribe/latest/dg/how-alternatives.html

Reply:
those say that the account id and job id are part of the output schema.  Are we removing those?

Reply:
Yes, the call-api removes that.
From the `com.bandwidth.voice.http.api.model.Transcription` object we take only the `results` property and that is what is sent to the customer.